package/MDAnalysis/coordinates/PDB.py

```diff
@@ -651,7 +700,24 @@ def _write_pdb_header(self):
             self.REMARK(*remarks)
         except AttributeError:
             pass
-        self.CRYST1(self.convert_dimensions_to_unitcell(u.trajectory.ts))
+
+        if u.dimensions is not None:
```

Comment:
Check for `[0.0, 0.0, 0.0, 0.0, 0.0, 0.0]`?

Reply:
Question here (and also relates to the above removal of the reference to cryo-em), are the default only [0,0,0,0,0,0] and [1,1,1,90,90,90] or are there more? If it's the latter case, then we might have to start handing things by remark number.

Reply:
Good point. Maybe this should be addressed in a different place? Here I could check for both `None` and the default `[0.0, 0.0, 0.0, 0.0, 0.0, 0.0]`

Reply:
Yeah, I'd vote for that. Either way, issues should probably be opened for further changes to the behaviour of PDBReader.

Reply:
Opened #2698 for meaningless/unspecified cell dimensions consistency.